packages/common/src/api/tan-query/useCollection.ts

```diff
@@ -37,6 +38,6 @@ export const useCollection = (
       return await batchGetCollections.fetch(collectionId!)
     },
     ...options,
-    enabled: options?.enabled !== false && !!collectionId
```

Comment:
technically the fix could be `&& collectionId` but that is a lot more subtle, so figured I'd be explicit

Reply:
Nice! Mind adding this to useTracks useCollections and useUsers too?

Could also add filters in the batcher to be extra safe